ports/espressif/boards/adafruit_feather_esp32s2/pins.c

```diff
@@ -17,6 +17,7 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_D6), MP_ROM_PTR(&pin_GPIO6) },
 
     { MP_ROM_QSTR(MP_QSTR_I2C_POWER_INVERTED), MP_ROM_PTR(&pin_GPIO7) },
+    { MP_ROM_QSTR(MP_QSTR_I2C_POWER), MP_ROM_PTR(&pin_GPIO7) },
```

Comment:
I agree entirely it's confusing as-is. I think referencing revs A/B makes more sense because they will be the more finite supply in the end - e.g. eventually most folks will be using rev C or later.

Maybe `I2C_POWER_INVERTED_REV_A_B`. Or the first one @dhalbert  suggested also works.

Reply:
i would rather get rid of INVERTED altogether and just provide a functional example that works with either

Reply:
> i would rather get rid of INVERTED altogether and just provide a functional example that works with either

So check the initial value before changing it and assume that state means "on", I guess. If you want to turn it off, toggle it.

Reply:
exactly